src/variation/utils.py

```diff
@@ -237,3 +237,21 @@ def get_vrs_loc_seq(
     else:
         ref = None
     return ref or None  # get_reference_sequence can return empty str
+
+
+def get_copy_change(efo_code: models.CopyChange) -> MappableConcept:
```

Comment:
I think `get_copy_change_concept` or `get_copy_change_object` would be a little less ambiguous